test/sharness/t0100-name.sh

```diff
@@ -225,6 +225,15 @@ test_name_with_self() {
         grep "argument \"ipfs-path\" is required" curl_out
         '
 
+        # test publishing with TTL
+        test_expect_success "'ipfs name publish --ttl=30m' succeeds" '
+        ipfs name publish --ttl=30m --allow-offline "/ipfs/$HASH_WELCOME_DOCS" && 
+        ipfs routing get "/ipns/$PEERID" > ipns_record &&
+        ipfs name verify-record $PEERID < ipns_record > verify_output &&
+        test_should_contain "$HASH_WELCOME_DOCS" verify_output
+        test_should_contain "1800000000000" verify_output
+        '
```

Comment:
I'm not extremely happy with this test because I call a lot of commands, so many things can fail. Do you have any suggestion? I could split them in 3 tests: one per command.